addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComTemperatureHumidityMessage.java

```diff
@@ -90,7 +109,7 @@ public byte toByte() {
 
     private final static List<RFXComValueSelector> supportedOutputValueSelectors = Arrays.asList();
 
-    public SubType subType = SubType.UNDEF;
+    public SubType subType = SubType.TH1;
```

Comment:
Why are we selecting a default subtype instead of starting with unknown? Until the message has been encoded, this value could be incorrect (applies to a couple of places).